synapse/handlers/federation.py

```diff
@@ -101,6 +102,8 @@
 
 logger = logging.getLogger(__name__)
 
+soft_failed_event_counter = Counter("synapse_handlers_federation_soft_failed_event", "")
```

Comment:
also, please populate the help string (the second arg to `Counter`). I know not all of our existing code sets a good example here, but let's get it right for new code.